release-process.rst

```diff
@@ -64,65 +80,144 @@ Example time line with only one major version at a time
 
     **** pre release phase
     ++++ release lifetime with all bug fixes, no feature addition
-    ---- release lifetime security  fixes only
+    ---- release lifetime security fixes only
+    G    GA Release
     D    EOL
+
     Version Time ->
-           2011        2012       2013         2014        2015        2016        2017
+           2023        2024       2025         2026        2027        2028        2029
             |     |     |     |     |     |     |     |     |     |     |     |     |
-    5.3     +++++++++++++-----D
-    5.4     |*****+++++++++++++++++++++++++-----------D
-    5.5     |     |     |******++++++++++++++++++++++++-----------D
-    5.6     |     |     |     |     |******++++++++++++++++++++++++-----------D
-    6.0     |     |     |     |                 |******++++++++++++++++++++++++-----------D
-    6.1     |     |     |     |                             |******++++++++++++++++++++++++-----------D
+    8.1     |++++++++++-------------------------D
+    8.2     |+++++++++++++++++++++++------------------------D
+    8.3     |     *****G++++++++++++++++++++++++------------------------D
+    8.4     |     |     |     |****G++++++++++++++++++++++++------------------------D
 
-Example time line with two majors versions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-However it could happen that a new major version is desired while the active
-major version is still heavily used. Like what we had between php 4 and 5 or
-for the oldest, between php 3 and 4.
+Release Timeline
+================
 
-::
+The process starts the first Tuesday of July of each year, and nominally runs
+for 20 weeks. With 3 alpha releases, 3 beta releases, 4 release candidates,
+and a GA (x.0.0) release.
 
-    **** pre release phase
-    ++++ release lifetime bugs
-    ---- release lifetime security only
-    D    EOL
-    Version Time ->
-           2011        2012       2013         2014        2015        2016        2017
-            |     |     |     |     |     |     |     |     |     |     |     |     |
-    5.3     +++++++++++++-----D
-    5.4     |*****+++++++++++++++++++++++++-----------D     |     |     |     |     |
-    5.5     |     |     |******++++++++++++++++++++++++-----------D     |     |     |
-    5.6     |     |     |           |******++++++++++++++++++++++++-----------D
-    6.0     |     |     |******++++++++++++++++++++++++-----------D     |     |
-    6.1     |     |     |           |******++++++++++++++++++++++++-----------D
+Examples are given for 2024 and PHP 8.4. Releases are tagged on the Monday of
```

Comment:
Our [existing release guidelines](https://github.com/php/php-src/blob/master/docs/release-process.md#general-notes-and-tips) are to package/tag "two days before a release" and explicitly call out Tuesday. Do we want to change _that_ guidance to be in alignment with "the Monday of each week" or revisit this callout instead?

Reply:
No. I got this one wrong. And that explains why I asked for reviews :-D.